tools/pr.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+
+import subprocess
```

Comment:
This file could do with a comment explaining how to use this script and what it does.